### DISCUSSION TITLE
Does it make sense to run the initial test run when we have no changes?

### DISCUSSION THREAD

**Discussion by phmonte** [stryker-mutator/stryker-net#2376] [General]
Is it possible to cancel the execution of Initial testrun when we have no changed files?

[14:58:47 INF] 0 files changed
[14:58:47 INF] Total number of tests found: 979.
[14:58:47 INF] Initial test run started.

If we don't have changed files, is there any reason to run the Initial testrun?

**Comment by rouke-broersma** [comment]
Sorry for missing this, good suggestion 👍
